airgun/widgets.py

```diff
@@ -2181,7 +2181,7 @@ class PopOverWidget(View):
     """
 
     ROOT = '.'
-    column_value = Text(".//span[contains(@class, 'editable')]")
+    column_value = Text(".//div[@class='ellipsis editable']")
```

Comment:
![image](https://user-images.githubusercontent.com/3190629/149810165-50354929-407a-478d-89ba-b3405ba7bbc7.png)

A Better Locator would be using contains as there are empty ones. e.g. $x("//div[contains(@class,'ellipsis editable')]")